I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -4859,6 +4971,108 @@ module ietf-optical-impairment-topology {
    read-only; however, these data nodes may be considered sensitive or
    vulnerable in some network environments (TBD).
   </t>
+-->
+
+  <t>
+   The YANG module specified in this document defines a schema for data
+   that is designed to be accessed via network management protocols
+   such as NETCONF <xref target="RFC6241"/> or RESTCONF
+   <xref target="RFC8040"/>. The lowest NETCONF layer is the secure
+   transport layer, and the mandatory-to-implement secure transport is
+   Secure Shell (SSH) <xref target="RFC6242"/>. The lowest RESTCONF
+   layer is HTTPS, and the mandatory-to-implement secure transport is
+   TLS <xref target="RFC8446"/>.
+  </t>
+  
+  <t>
+   The Network Configuration Access Control Model (NACM)
+   <xref target="RFC8341"/> provides the means to restrict access for
+   particular NETCONF or RESTCONF users to a preconfigured subset of
+   all available NETCONF or RESTCONF protocol operations and content.
+  </t>
+  
+  <t>
+   The YANG module specified in this document imports the ietf-network
+   and ietf-network-topology models defined in
+   <xref target="RFC8345"/>. This YANG module augments te-topology
+   defined in <xref target="RFC8795"/> and consequently inherits
+   te-topology subtrees and data nodes and their potential
+   sensitivities and vulnerabilities. Therefore, the security
+   considerations from <xref target="RFC8345"/> also apply to the YANG
+   module defined in this document.
+  </t>
+  
+  <t>
+   The data nodes in this YANG module are not writable nor creatable
+   nor deletable (i.e., config true) the data nodes are readable only
```

Comment:
"(i.e. not config true)" would be clearer. A semicolon after the bracket would be welcome.